### PR TITLE
[DOXFIX] delete System 

### PR DESCRIPTION
**Pull request by silvaco**
change "system configuration" to "configuration"
https://alluxio.atlassian.net/browse/ALLUXIO-2009?filter=-4

### CODE REVIEW

File: docs/cn/Web-Interface.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Alluxio master 主页如下所示:
 
 ## 配置页面
 
-查看当前系统的配置信息，点击屏幕上方导航栏的"System Configuration"。
+查看当前系统的配置信息，点击屏幕上方导航栏的"Configuration"。
```
Comment (ifcharming) [inline docs/cn/Web-Interface.md]: why do you want to delete "System" ?
The actual Web UI has "System Configuration" navigation bar.

**Comment by ifcharming** [comment]
thanks @silvaco for the contribution.
Can you please explain a bit more about this PR?

File: docs/cn/Web-Interface.md (hunk 1 above)
Comment (ifcharming) [inline docs/cn/Web-Interface.md]: oh I see, the imagine in the doc is not updated yet...

**Comment by apc999** [comment]
LGTM. Thanks for the patch!

**Pull request merged**